roles/setup_selfsigned_cert/meta/main.yml

```diff
@@ -1,2 +1,2 @@
-dependencies:
-  - role: validate_inventory
+#dependencies:
+#  - role: validate_inventory
```

Comment:
Would it work by adding a when clause to the `role` that way we don't break crucible functionality.

Reply:
sounds good to me. I'll work on it.

Reply:
OK, so after checking the code in the validate_inventory role we saw we can set the inventory_validated variable to true so the inventory validation is skipped.
So the current solution is working without having to change anything from this repo side.
We still need to get the "ignore boot device" setting change merged, though.